### DISCUSSION TITLE
Your benefits will be available soon

### DISCUSSION THREAD

**Discussion by AshokChaudhary1** [community/community#144775] [GitHub Education] [answered]
### Hi everyone,

Sir please accept my request, My exams are coming in December, please accept it soon, I have to do a lot of projects
![IMG_20241116_190253](https://github.com/user-attachments/assets/880c589e-2bb4-4dd5-b07e-9cf987a483de)

Thank you sir , I am waiting please accept my request 🥺🥺

**Comment by davevad93** [comment, marked as answer]
Hi @AshokChaudhary1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AshokChaudhary1 was approved and your benefits are now available.`

GitHub no longer provides manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

Don't worry you'll have your benefits soon.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
